recipes/ace_cream/meta.yaml

```diff
@@ -0,0 +1,84 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is conviniet
+# when copying and paster from another recipe, but not really needed.
+{% set name = "ace_cream" %}
+{% set version = "0.4.post8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/zhaofeng-shu33/ace_cream.git
```

Comment:
Please install from a tarball, not from git:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos